### PR TITLE
Fixed bringToTop bug for Windows

### PR DESCRIPTION
**Pull request by arejaycola**
Fixes issue #36 with window.bringToTop() not working properly. Followed advice from @stefnotch.

### CODE REVIEW

File: lib/windows.cc [hunk 1]
```diff
@@ -258,11 +258,19 @@ Napi::Boolean showWindow (const Napi::CallbackInfo& info) {
 
 Napi::Boolean bringWindowToTop (const Napi::CallbackInfo& info) {
     Napi::Env env{ info.Env () };
-
     auto handle{ getValueFromCallbackData<HWND> (info, 0) };
     BOOL b{ SetForegroundWindow (handle) };
```
Comment (stefnotch) [inline lib/windows.cc:262]: This line also needs to be removed. (See below)

File: lib/windows.cc [hunk 2]
```diff
@@ -258,11 +258,19 @@ Napi::Boolean showWindow (const Napi::CallbackInfo& info) {
...
-    SetActiveWindow (handle);
-    SetWindowPos (handle, HWND_TOP, 0, 0, 0, 0, SWP_SHOWWINDOW | SWP_NOMOVE | SWP_NOSIZE);
+
+    HWND hCurWnd = ::GetForegroundWindow();
+    DWORD dwMyID = ::GetCurrentThreadId();
+    DWORD dwCurID = ::GetWindowThreadProcessId(hCurWnd, NULL);
+    ::AttachThreadInput(dwCurID, dwMyID, TRUE);
+    ::SetWindowPos(handle, HWND_TOPMOST, 0, 0, 0, 0, SWP_NOSIZE | SWP_NOMOVE);
+    ::SetWindowPos(handle, HWND_NOTOPMOST, 0, 0, 0, 0, SWP_NOSIZE | SWP_NOMOVE);
+    ::SetForegroundWindow(handle);
```
Comment (stefnotch) [inline lib/windows.cc:270]: This line returns a boolean which needs to be assigned to b.

Comment (arejaycola) [reply lib/windows.cc:270]: Sorry what line are you referring to?

Comment (stefnotch) [reply lib/windows.cc:270]: `::SetForegroundWindow(handle);`

**Comment by cannedconcept** [comment]
I still seem to have this issue with v2.2.4 on win10.

Dirty workaround is  window.restore(); first but that changes the window size as well.

**Pull request merged**